src/app/admin/admin-registries/bitstream-formats/bitstream-formats.component.ts

```diff
@@ -33,25 +33,26 @@ export class BitstreamFormatsComponent implements OnInit, OnDestroy {
    * The current pagination configuration for the page used by the FindAll method
    * Currently simply renders all bitstream formats
    */
-  config: FindListOptions = Object.assign(new FindListOptions(), {
-    elementsPerPage: 20
-  });
+  // config: FindListOptions = Object.assign(new FindListOptions(), {
+  //   elementsPerPage: 20
+  // });
```

Comment:
Small thing, can we just remove this unused code?  It doesn't appear there is a reason to keep this around. (and the comment just above it).

Reply:
Hi @yingjin 
I'm still working with the page layout, greetings!
![image](https://user-images.githubusercontent.com/30701057/198724564-cb305660-3666-4e5f-a4b4-6a195021adb2.png)